09-features.md

```diff
@@ -29,7 +29,12 @@ These flags may only be used in the `init` message:
 
 ## Assigned `globalfeatures` flags
 
-There are currently no `globalfeatures` flags.
+These flags may only be used in the `init` and `node_announcement` message:
+
+| Bits | Name             |Description                                     | Link                                                                |
+|------|------------------|------------------------------------------------|---------------------------------------------------------------------|
+| 14/15 | `option_support_large_channel` | Can create large channels		   | [Rationale](#Rationale) |
```

Comment:
I think that link is broken, since Rationale is in a separate BOLT?

Reply:
Isn't it below at line 45? [Rationale](https://github.com/araspitzu/lightning-rfc/blob/large_channel_support/09-features.md#rationale)